app/src/main/java/ru/semper_viventem/pixel4scanner/ShareUtils.kt

```diff
@@ -10,13 +10,15 @@ import java.io.File
 
 private const val DIRECTORY = "sharing_files/"
 
-fun shareBitmapAsImage(activity: Activity, depthMap: Bitmap, photo: Bitmap) {
+fun shareBitmapAsImage(activity: Activity, depthMap: Bitmap, photo: Bitmap, json : String) {
```

Comment:
This function for Image exporting. Can we create separated function like a `shareXYZAsFile` and use it for share your json?

Reply:
Hi sorry for the slow reply.
The problem I see with a second function is that we will need a second button to share the XYZ too.
Can be OK but make the application more complex for a demo.
Maybe just rename the function "Share" or "ShareStuff"?